asciidoctorj-api/build.gradle

```diff
@@ -0,0 +1,20 @@
+apply plugin: 'osgi'
+
+dependencies {
+}
+
+
+javadoc {
+  classpath = sourceSets.main.output + sourceSets.main.compileClasspath + project(':asciidoctorj-test-support').sourceSets.test.output
```

Comment:
I think we don't need the asciidoctorj-test-support project for the java docs, right?

Reply:
This is probabelly a copy paste error (I am not really a gradle user). What would be the correct value? nothing? Do I need a javadoc section?

Reply:
I guess this line can be completely removed. For this project we shouldn't need any particular classpath.

We would need this module though to create a complete javadoc for the API in asciidoctorj-api and asciidoctorj itself, but we can check that later.